ui/app/adapters/role.js

```diff
@@ -6,7 +6,17 @@
 // @ts-check
```

Comment:
I don't know how the license update affects you here?

Reply:
I need the license here, and for ts-check to come afterward. ts-check is some type safety in JS.

Reply:
I meant the BSL  on top of the page :)